src/main/java/com/wimbli/WorldBorder/WorldFillTask.java

```diff
@@ -249,7 +249,8 @@ public void run()
 			if (!chunkOnUnloadPreventionList(unload.x, unload.z))
 			{
 				world.setChunkForceLoaded(unload.x, unload.z, false);
-				world.unloadChunkRequest(unload.x, unload.z);
```

Comment:
Use `Config.AvailableMemoryTooLow()` to check if memory is low before calling this loop instead of removing this line.